evaluation/evaluator.go

```diff
@@ -311,6 +312,12 @@ func (e Evaluator) checkPreRequisite(fc *rest.FeatureConfig, target *Target) (bo
 	return true, nil
 }
 
+// public wrapper for the evaluate.
+func (e Evaluator) Evaluate(identifier string, target *Target, kind string) (rest.Variation, error) {
```

Comment:
Could you add a function comment per the linter suggestion.  Anything thats public should have a comment along with it.

Reply:
noted!